docs/modules/ROOT/partials/getting-started/con-registry-serdes-strategy.adoc

```diff
@@ -4,18 +4,30 @@
 [id='registry-serdes-concepts-strategy-{context}']
 = Strategies to lookup a schema
 
-The Kafka client serializer/deserializer uses a lookup _strategy_ to determine the artifact ID or global ID under which the message schema is registered in {registry}.
+The Kafka client serializer uses two lookup _strategies_ to determine the artifact ID and global ID under which the message schema is registered in {registry}.
 
-For a given topic and message, you can use implementations of the following Java classes:
+For a given topic and message, you can use implementations of the following Java interfaces:
 
 * `ArtifactIdStrategy` to return an artifact ID
 * `GlobalIdStrategy` to return a global ID
 
-The artifact ID returned depends on whether the _key_ or _value_ in the message is being serialized.
+The artifact ID strategy provides a way to map the Kafka topic and message information to the ID of an artifact in
+{registry}.  The common convention for the mapping is to combine the Kafka topic name with either `key` or `value`
+(depending on whether the serializer is being used for the Kafka message key or value).  However, alternative
+conventions can be used for the mapping, either by using an alternative strategy provided by {registry} or by
+creating a custom Java class that implements `io.apicurio.registry.utils.serde.strategy.ArtifactIdStrategy`.
+
+The global ID strategy is responsible for locating and identifying the specific *version* of the schema registered
```

Comment:
Suggest adding block heading to help break up the text, for example: ".Global ID strategy"